src/opower/opower.py

```diff
@@ -189,6 +189,19 @@ def __init__(
         self.customers: list[Any] = []
         self.user_accounts: list[Any] = []
 
+    def _get_utilitycode(self) -> str:

```

Comment:
Do you need this? Can't you replace the caller with `self.utility.utilitycode()`?

Reply:
I'd remove this and just call `self.utility.utilitycode()`. And have `utilitycode` return the domain by default in the base.